FHEM/00_SIGNALduino.pm

```diff
@@ -4654,6 +4714,33 @@ When set to 1, the internal "RAWMSG" will not be updated with the received messa
 		<li>rawmsgEvent<br>
 		Bei der Einstellung "1", l&ouml;sen empfangene Rohnachrichten Ereignisse aus.
 		</li><br>
+		<a name="rfmode"></a>
```

Comment:
Vermutlich stammt das ja vom cc usb lite, der cc1101 quasi mit im Namen trägt. Korrekterweise ist das ein cc1101 spezifisches Attribut und ich fänd es gut, wenn das attribut, wie auch die set / get Befehle mit cc Beginnen würden.

Und da es Register setzt käme ich auf eine Bezeichnung

`cc_RegRF `oder sowas in der Richtung.

Reply:
https://wiki.fhem.de/wiki/Rfmode

Ja, die Umstellung erfolgt im cc1101 aber den Empfänger als einzelnes betrachtet, wird der Mode umgestellt, daher Rfmode.